app/src/main/java/com/sdp13epfl2021/projmag/model/ImmutableProfile.kt

```diff
@@ -12,13 +12,14 @@ data class ImmutableProfile private constructor(
 ) :
     Parcelable, Serializable {
 
-
     companion object {
         private const val MAX_LAST_NAME_SIZE = 40
         private const val MAX_FIRST_NAME_SIZE = 40
         private const val MAX_AGE = 120
         private const val MIN_AGE = 15
 
+        val EMPTY_PROFILE = ImmutableProfile("", "", 0, Gender.OTHER, 0, "", Role.OTHER)
+
```

Comment:
Maybe add an .EMPTY/.UNDEFINED to differentiate OTHER from something that hasn't been defined yet